lib/archive.ml

```diff
@@ -6,6 +6,20 @@
 
 open Bos_setup
 
+(* Safe wrapping for some Bytes functions *)
+
+module Sbytes = struct
+  include Bytes
```

Comment:
I'm not too fond of such includes. I don't think they're necessary unless we really want to shadow the whole module or stdlib with global opens, which I don't fancy too much either.

I think here, it adds an indirection layer and makes it hard to know which functions are actually being used.

What do you think?

Reply:
I agree, done